clients/filter/filter.go

```diff
@@ -92,11 +96,16 @@ func (c *Client) GetOutput(ctx context.Context, filterOutputID string) (m Model,
 		return
 	}
 
+	common.AddServiceTokenHeader(req, serviceAuthToken)
```

Comment:
Same her: You need to set the provided context on the request otherwise it will be created with it's own which will not have any of the traceID's and other good stuff we store in it.

```
req = req.WithContext(ctx)
```